update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -95,19 +103,26 @@ public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pa
 									gkDOI.getAttributeValuesList("modified");
 									gkDOI.addAttributeValue("modified", instanceEditGK);
 									gkDOI.setAttributeValue("doi", updatedDoi);
-									dbaGkCentral.updateInstanceAttribute(gkDOI, "modified");
-									dbaGkCentral.updateInstanceAttribute(gkDOI, "doi");
+									if (!testMode) {
```

Comment:
Your code will still perform a commit at the end, eve if `testMode` is `true`. Is there a possibility that something might get committed in testMode? Maybe there should be a check for `testMode` at the end, and perform a rollback if `testMode` is `true`.

Reply:
This was a good point, I needed to actually do a test mode condition around the 'createInstanceEdit' calls above